tests/README.rst

```diff
@@ -45,7 +47,6 @@ For CrateDB >= 1.2 you would need to change the following setting in your docker
 
   >>> docker-machine ssh default "sudo sysctl -w vm.max_map_count=262144"
 
-The tests are run using the `zope.testrunner <https://pypi.python.org/pypi/zope.testrunner>`_.
 To run the tests::
 
-  >>> ./bin/test
+  >>> python3 tests/tests.py
```

Comment:
Is it possible to run the tests using `python -m unittest` ?

Reply:
That will result in unittest attempting to discover classes that extend TestCase. In our case these are in itests.py . The module's logic will instantiate them test cases bypassing the intended constructor (which passes the docker_layer, instead of a str). This results in attempts to get attributes out of a str, which fails.

Reply:
See the other command about the test discovery protocol.